wurf_runner.py

```diff
@@ -244,9 +244,19 @@ def assemble_output(stdout, stderr):
     """Helper function to assemble output message from the test results"""
     msg = []
     if stdout:
+        try:
+            stdout.encode("ascii")
+        except UnicodeError:
+            stdout = stdout.encode("utf-8")
         msg.append('\nstdout:\n\n{}'.format(stdout))
```

Comment:
We could go for utf-8 as default as well I guess. :)

I just thought it was annoying that the wurf-runner crashes if it tries to output a non ascii character.